### PR TITLE
[PIM-6979] Fix Styleguides for v2

### PR DESCRIPTION
**Pull request by pierallard**
Includes:
- the "nono design" for the styleguides
- all the updates we made from 1.7 to 2.0 for the LESS files

### CODE REVIEW

File: app/cache/dev/annotations/36cac57fbb6eea8ce8ea0ff2b3aa69ef4bf44224.cache.php [hunk 1]
```diff
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
```
Comment (tamarasaurus) [inline app/cache/dev/annotations/36cac57fbb6eea8ce8ea0ff2b3aa69ef4bf44224.cache.php]: .gitignore app/cache ?

File: design_pim/styleguide/index.php [hunk 2]
```diff
@@ -22,129 +23,150 @@
...
-<script type="text/javascript" src="../_static/doctools.js"></script>
-<script type="text/javascript" src="../_static/js/theme.js"></script>
-<script type="text/javascript">
-    jQuery(function () {
-        SphinxRtdTheme.StickyNav.enable();
+
+<script>
+    $(function(){
+        var navbar = $('.navbar');
+        $(window).scroll(function(){
```
Comment (tamarasaurus) [inline design_pim/styleguide/index.php]: Why is this needed?

Comment (pierallard) [reply design_pim/styleguide/index.php]: Displays a shadow on main navbar
